src/index.js

```diff
@@ -0,0 +1,125 @@
+let ArrayList = {
+    array: [],
+    size: 0,
+    init: function(argument) {
+        if (!argument) return 'Please input an array';
+        ArrayList.array = argument;
+        ArrayList.size = argument.length;
+
+        return ArrayList.array
+    },
+    addAtEnd: function(arrElm) {
+        if (!arrElm)  return 'Please input an array';
+            const arr = ArrayList.array;
+            arr[arr.length] = arrElm;
+            ArrayList.size = arr.length;
+
+            return arr;
+    },
+    clearObject: function() {
+        ArrayList.array = [];
+        ArrayList.size = 0;
+
+        return ArrayList.array
+    },
+    clearFromEnd: function() {
+        const arr = ArrayList.array;
+        arr.length = [arr.length - 1];
+        ArrayList.size = arr.length;
+        
+        return ArrayList.array;
+    },
+    deleteFirst: function() {
+        const arr = ArrayList.array;   
+        let newArray = [];     
+
+        for (let i = 1; i < arr.length; i++) {
+            newArray[i-1] = arr[i];
+        };
+        ArrayList.array = newArray;
+        ArrayList.size -= 1;
+
+        return arr[0];
+    },
+    addToFirst: function(argument) {
+        if (!argument) return 'Please check your input';
+        const arr = ArrayList.array;
+        let newArr = [argument];
+
+        for (let i = 0; i < arr.length; i++) newArr[i + 1] = arr[i]
```

Comment:
It's good that you trying to write shorter as much it possible but more readable it looks in braces, I mean for(){}